src/back/interfaces/handlers.py

```diff
@@ -0,0 +1,17 @@
+"""Module containing handlers' interfaces."""
+from typing import Protocol, Self
+
+from tornado.iostream import IOStream
+
+from src.back.interfaces.message import MessageInterface
+from src.back.interfaces.room_registry import RoomRegistryInterface
+
+
+class CreateRoomHandlerInterface(Protocol):
+    """Interface for create room."""
+
+    room_registry: RoomRegistryInterface
+
+    def handle(self: Self, message: MessageInterface, stream: IOStream) -> None:
+        """Create room."""
+        raise NotImplementedError
```

Comment:
We don't need concrete interfaces for the same concrete implementations. Just:
```python
class MessageHandler(Protocol):
    """Message handler interface."""

    def handle(self: Self, message: MessageInterface, stream: IOStream) -> None:
        """Handle message."""
        raise NotImplementedError
```

Reply:
And instead of `IOStream` use `WriteStreamInterface`. This way we'll make sure that handlers don't try to read from the stream.